### ISSUE TITLE
Show account's balance and related transactions 

### ISSUE THREAD

**Issue by Max-Pano** [jpmorganchase/cakeshop#51]
I wonder if you have plans to add the widget of account balance query like Etherscan do.
Certain feature is badly needed for our quorum implementation. 😄

**Comment by ncioj10** [comment]
Yes we would need that as weel, because it is vital for us to have a account balance overview as well as a transactions overview.
